pkg/api/types.go

```diff
@@ -738,7 +771,35 @@ type Endpoints struct {
 	TypeMeta   `json:",inline"`
 	ObjectMeta `json:"metadata,omitempty"`
 
-	Endpoints []string `json:"endpoints,omitempty"`
+	Endpoints []Endpoint `json:"endpoints,omitempty"`
```

Comment:
Is there really a fan out for a service for that?  4000 things talking to any of 4000 other things?


> On Feb 14, 2015, at 5:06 PM, Tim Hockin <notifications@github.com> wrote:
> 
> In pkg/api/types.go:
> 
> > @@ -738,7 +771,35 @@ type Endpoints struct {
> >  	TypeMeta   `json:",inline"`
> >  	ObjectMeta `json:"metadata,omitempty"`
> >  
> > -	Endpoints []string `json:"endpoints,omitempty"`
> > +	Endpoints []Endpoint `json:"endpoints,omitempty"`
> @smarterclayton I don't think O(hundreds) or even O(thousands) is out of line with reality. if someone really wants to scale and finds that 4000 1-core jobs is better than 500 8-core jobs, who are we to say that's wrong. Lots of stuff will have to be smoothed out to get that far, of course :)
> 
> To Brian, I ACK the eventual need, but I don't want to trailblaze this area as a predicate to getting v1b3 out.
> 
> —
> Reply to this email directly or view it on GitHub.
>